plugins/window_size/lib/src/window_size_channel.dart

```diff
@@ -74,6 +82,12 @@ const String _getWindowMinimumSizeMethod = 'getWindowMinimumSize';
 /// unconstrained in that dimension.
 const String _getWindowMaximumSizeMethod = 'getWindowMaximumSize';
 
+/// The method name to get the window title of a window.
+const String _getWindowTitleMethod = 'getWindowTitle';
```

Comment:
Do you have a use case for a getter? The closest Flutter system channel I can think of is `setApplicationSwitcherDescription`, which doesn't have a getter, so unless there's a specific need for this I would prefer to not add it speculatively.

Reply:
Dropped